tests/e2e/test_save.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+import torch
+
+from azstoragetorch.io import BlobIO
+
+
+@pytest.fixture(scope="module", autouse=True)
+def torch_hub_cache(tmp_path_factory):
+    current_dir = torch.hub.get_dir()
+    torch.hub.set_dir(tmp_path_factory.mktemp("torch_hub"))
+    yield
+    torch.hub.set_dir(current_dir)
+
+
+@pytest.fixture(scope="module")
+def model():
+    model = torch.hub.load("pytorch/vision:v0.10.0", "resnet101", pretrained=False)
+    return model
```

Comment:
It probably makes sense to move both of these fixtures to the `conftest.py` file so that we can reuse it between the `test_save.py` and `test_load.py` files. We'll probably have to expand the scope to package for these as well.

Another option that I like as well is if we combine the `test_save.py` and `test_load.py` into a `test_torch_serialize.py` file where we generally test integration with pytorch serialization methods. I'm fine with either option.